Tests/DangerTests/GitTests.swift

```diff
@@ -1,14 +1,17 @@
 import XCTest
 import Danger
 
-class GitTests: XCTestCase {
+final class GitTests: XCTestCase {
```

Comment:
does this final really add any value?

Reply:
Even if minimum it should improve the performances because the function call doesn't go through the virtual dispatch.